index.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 
 var defined = require('defined');
+var through = require('@ljharb/through');
+
 var createDefaultStream = require('./lib/default_stream');
 var Test = require('./lib/test');
 var Results = require('./lib/results');
-var through = require('@ljharb/through');
 
 var canEmitExit = typeof process !== 'undefined' && process
+	// @ts-expect-error i think old browserify uses `process.browser`
 	&& typeof process.on === 'function' && process.browser !== true;
 var canExit = typeof process !== 'undefined' && process
 	&& typeof process.exit === 'function';
 
+/** @typedef {import('.')} Tape */
+/** @typedef {ThisParameterType<Tape>} Harness */
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/issues/7670: it's probably best to export `Harness` standalone the way `HarnessConfig` is. If folks are able to refer to it (which they can by way of `this: tape.Harness`, they're going to want to be able to have type annotations refer to it easily.

Reply:
This is already done; i'd just not updated this usage site. Done now.